<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="plotWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="controlWidget" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>171</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QPushButton" name="SpanBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Freq</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CenterFreqBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>End Freq</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Sizebtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Step Size</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Center : </string>
    </property>
   </widget>
   <widget class="QLabel" name="center_freq_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="span_label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>530</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>530</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Span :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RBW    :</string>
    </property>
   </widget>
   <widget class="QLabel" name="rbw_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>550</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="swt_label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>550</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>550</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SWT  :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Step : </string>
    </property>
   </widget>
   <widget class="QLabel" name="size_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>530</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
